seatunnel-ci-tools/src/test/java/org/apache/seatunnel/api/ConnectorOptionCheckTest.java

```diff
@@ -0,0 +1,149 @@
+                                                                                    .toString()
+                                                                                    .replace(
+                                                                                            JAVA_FILE_EXTENSION,
+                                                                                            "")
+                                                                                    .concat(
+                                                                                            "Options"));
+                                                                }
+                                                            });
+                                                    NodeList<ClassOrInterfaceType> extendedTypes =
+                                                            classDeclaration.getExtendedTypes();
+                                                    extendedTypes.forEach(
+                                                            extendedType -> {
+                                                                if (extendedType
+                                                                        .getNameAsString()
+                                                                        .equals(
+                                                                                "AbstractSimpleSink")) {
+                                                                    connectorOptionFileNames.add(
+                                                                            path.getFileName()
+                                                                                    .toString()
+                                                                                    .replace(
+                                                                                            JAVA_FILE_EXTENSION,
+                                                                                            "")
+                                                                                    .concat(
+                                                                                            "Options"));
+                                                                }
+                                                            });
+                                                }
+                                            });
+                        } catch (IOException e) {
+                            throw new RuntimeException(e);
+                        }
+                    });
+            connectorClassPaths.forEach(
+                    path -> {
+                        String className =
+                                path.getFileName().toString().replace(JAVA_FILE_EXTENSION, "");
+                        connectorOptionFileNames.remove(className);
+                    });
+            Assertions.assertEquals(
```

Comment:
Can we add some white list to let the test case passes? Then when one connector had Options class, we can remove it from white list.

Reply:
Ok, I will add the connector under this issue to the whitelist, https://github.com/apache/seatunnel/issues/8576.